src/schema/v2/article/models.ts

```diff
@@ -0,0 +1,85 @@
+import { ResolverContext } from "types/graphql"
+import {
+  GraphQLEnumType,
+  GraphQLID,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { ImageType } from "../image"
+import uuid from "uuid/v5"
+
+export const ArticleImageSection = new GraphQLObjectType<any, ResolverContext>({
+  name: "ArticleImageSection",
+  isTypeOf: ({ type }) => {
+    return type === "image"
+  },
+  fields: {
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      resolve: ({ url }) => {
+        return uuid(url, uuid.URL)
+      },
+    },
```

Comment:
This is the only slightly odd thing about this PR. I wanted all the `figures` to have IDs because this dramatically simplifies how we can implement the zoom gallery view since we know which ID we click on. Here I'm just constructing an ID based on the URL of the image.